database-plugins/src/main/java/co/cask/hydrator/plugin/db/batch/source/DBSource.java

```diff
@@ -136,6 +140,17 @@ public Schema getSchema(GetSchemaRequest request,
     }
   }
 
+  private static String removeConditionsClause(String importQuerySring) {
+    if (importQuerySring.toUpperCase().contains("WHERE $CONDITIONS AND")) {
```

Comment:
What happens if there are two spaces between $CONDITIONS and AND. Can you please make this robust.